docs/en/cloud/reference/cloud-compatibility.md

```diff
@@ -6,7 +6,7 @@ title: Cloud Compatibility
 
 # ClickHouse Cloud — Compatibility Guide
 
-This guide provides an overview of what to expect functionally and operationally in ClickHouse Cloud.
+This guide provides an overview of what to expect functionally and operationally in ClickHouse Cloud. While ClickHouse Cloud uses ClickHouse OSS, there may be some differences that exist due to the implementation and Control Plane in ClickHouse Cloud - you may find this blog on how we built ClickHouse Cloud interesting and relevant also: https://clickhouse.com/blog/building-clickhouse-cloud-from-scratch-in-a-year
```

Comment:
I like the addition. May I recommend small adjustments in wording?

> While ClickHouse Cloud is based on open-source ClickHouse distribution, there may be some differences in architecture and implementation. You may find this blog on how we built ClickHouse Cloud interesting and relevant to read as background: https://clickhouse.com/blog/building-clickhouse-cloud-from-scratch-in-a-year

Reply:
Sure, whatever you think best. I wasn't sure if "ClickHouse OSS" was the official name either. I think this paragraph will help people find this page more easily in search also - guessing they might be searching on "difference between ClickHouse and ClickHouse Cloud"